Formula/docker-clean.rb

```diff
@@ -0,0 +1,19 @@
+# Docker-Clean Script
+# Maintained by Killian Brackey && Sean Kilgarriff at ZZROT Design
+# Contact us at docker@zzrot.com for any issues or fork and pull from
+# the Github repo below.  Happy cleaning!
```

Comment:
Nobody will see these comments except for Homebrew maintainers and other contributors (like you) who will update this formula in the future. But those people will also see the `homepage` line below. Regular users are much more likely to consult `brew info docker-clean` or use `brew home docker-clean` to get to the right place to report upstream issues.